stable/ambassador/values.yaml

```diff
@@ -0,0 +1,126 @@
+# Default values for ambassador.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+daemonSet: false
+
+ambassador:
+  id: default
+
+namespace:
+  single: false
+  # name: default
+
+# Additional container environment variable
+env:
+  {}
+  # Exposing statistics via StatsD
+  # STATSD_ENABLED: true
+  # STATSD_HOST: statsd-sink
+
+image:
+  repository: quay.io/datawire/ambassador
+  # tag:
+  pullPolicy: IfNotPresent
+  # imagePullSecrets:
+
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  type: LoadBalancer
+
+  # Note that target http ports need to match your ambassador configurations service_port
+  # https://www.getambassador.io/reference/modules/#the-ambassador-module
+  http:
+    enabled: true
+    port: 80
+    targetPort: 80
```

Comment:
Yes, it would be possible. Requires default annotations for ambassador to listen on them tho, https://github.com/datawire/ambassador/pull/913/files#diff-37a8b369e38f567f14bf62d9263d31bbR39 but I think it would be fine to make this change here since ambassador will change default in a future release.